airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -229,7 +229,7 @@ class KubernetesPodOperator(BaseOperator):
     def __init__(
         self,
         *,
-        kubernetes_conn_id: str | None = None,  # 'kubernetes_default',
+        kubernetes_conn_id: str = "kubernetes_default",
```

Comment:
Actually, it’s probably better to use `KubernetesHook.default_conn_name` as the default instead.